packages/roosterjs-content-model/lib/modelApi/table/splitTableCellHorizontally.ts

```diff
@@ -39,6 +39,9 @@ export function splitTableCellHorizontally(table: ContentModelTable) {
                             cell.format
                         );
 
+                        //Keep the background color of the original cell
+                        newCell.dataset = { editingInfo: '{"bgColorOverride":true}' };
```

Comment:
Good finding!

But instead of just copy "editingInfo" and "bgColorOverride", you can actually copy all the dataset so that if we add more metadata in the future, they will also be cared.

```ts
newCell.dataset = {...cell.dataset};
```